### DISCUSSION TITLE
Spacy projects - multiple project.yml files

### DISCUSSION THREAD

**Discussion by lmanhes** [explosion/spaCy#7926] [Help: Best practices]
Hi,

I was wondering if it is possible to have several project.yml in the same repository or it is advised to create a different repository for each project ?

**Comment by svlandeg** [comment]
Hi! 

No, not as far as I know. The naming of the `project.yml` file is fixed. What you can do though, is create one repository for your project, and have two subdirectories in that, with each subdir having its own `project.yml` file.
